endstate_correction/analysis.py

```diff
@@ -86,50 +89,84 @@ def plot_results_for_equilibrium_free_energy(
     plt.close()
 
 
-def plot_endstate_correction_results(
-    name: str, results: Results, filename: str = "plot.png"
-):
+def return_neq_correction(
```

Comment:
Not really important, but maybe we want to name the function differently (`return_endstate_correction_results` ?) because `NEQ` is not the only option here

Reply:
good point, will update the name!